tools/accuracy/accuracy_checker.py

```diff
@@ -88,6 +88,12 @@ def parse_args():
                         default=False,
                         help='Turn on ort VERBOSE logging via session options')
 
+    parser.add_argument('--show-test-data',
+                        dest="show_data",
```

Comment:
probably want to clean this up elsewhere, but for now, we should try to be consistent between single and double quotes

Reply:
Used all single quotes